src/components/ResultsItemDetails.vue

```diff
@@ -86,4 +92,12 @@ export default {
     border-radius: 13%;
     padding: 1em;
   }
+
+   button.disabled {
+    background-color: darkgray;
+  }
+
+  .disabled {
+    pointer-events: none;
+  }
```

Comment:
Great ux/ui changes on this file.

Reply:
The feedback for the user is definitely coming along well now!